app/window.go

```diff
@@ -464,13 +464,31 @@ const (
 
 // WindowMode sets the window mode.
 //
-// Supported platforms are macOS, X11 and Windows.
+// Supported platforms are macOS, X11, Windows and JS.
 func WindowMode(mode wm.WindowMode) Option {
 	return func(opts *wm.Options) {
 		opts.WindowMode = &mode
 	}
 }
 
+const (
+	// UnlockedOrientation is the default behavior, allowing the rotate of the screen.
+	UnlockedOrientation = wm.UnlockedOrientation
+	// LockedPortrait locks the screen to portrait.
+	LockedPortrait = wm.LockedPortrait
+	// LockedLandscape locks the screen to landscape.
+	LockedLandscape = wm.LockedLandscape
+)
+
+// Orientation sets the orientation of the app.
+//
+// Supported platforms are Android and JS (JS may requires to run on Fullscreen).
+func Orientation(mode wm.OrientationMode) Option {
```

Comment:
It occurred to me that a function for enum options is redundant. Why not make the enums themselves Options?

```
var (
    AnyOrientation = orientation(wm.AnyOrientation)
   LandscapeOrientation = orientation(wm.LandscapeOrientation)
   ...
)

func orientation(mode wm.Orientation) Option
```

See https://gioui.org/commit/a06a7a4b3c107 for a similar change to WindowMode.
The same applies to WindowMode.